maas/testutils/helpers.go

```diff
@@ -1,9 +1,9 @@
 package testutils
 
 import (
+	"crypto/rand"
 	"fmt"
-	"log"
-	"math/rand"
+	mrand "math/rand"
```

Comment:
why the inclusion of two rand generators? is crypto proven to be more random?? why not just use that, rather than having both

Reply:
I tried to quickly update it but it seems that it requires some effort. Since it is a helper function for tests and we do not care for real randomness that much, I decided to come up with this format. @tobiasdemendonca is planning to cover all IP usages in tests with this function with a follow-up PR so he could try to include a full transition to `crypto/rand`.